docs/source/man.rst

```diff
@@ -204,7 +204,7 @@ Sources
 
 Use the ``qpc source`` command to create and manage sources.
 
-A source defines a collection of network information, including IP addresses or host names, or systems management solution information, in addition to SSH ports and SSH credentials. The SSH credentials are provided through reference to one or more credentials. A scan can reference a source so that the act of running the scan is repeatable, without a requirement to reenter network information for each scan attempt.
+A source defines a single entity or multiple entities to be inspected. A source can be a single physical machine, virtual machine, or container, or it can be a collection of network information, including IP addresses or host names, or systems management solution information such as vCenter Server or Satellite information. The source also contains information about SSH ports and SSH credentials that are needed to access the systems to be inspected. The SSH credentials are provided through reference to one or more of the Quipucords credentials that you configure. A scan  references a source so that the act of running the scan is repeatable, without a requirement to reenter the information for each scan attempt.
```

Comment:
"systems management solution information" sounds awkward?

How about "a systems management solution such as vCenter Server or Red Hat Satellite."

Reply:
Thanks for your comment, and I'll provide some background. qpc does two major things: gathers network information and gathers systems management solution information. There is some difference in function/required info when it does those two things, but some similarity between Sat/vC. "Systems management solution" was one of the more common terms that I saw as the generic noun for the type of management system that Satellite and vCenter Server are. See the following links where Satellite doc references itself as a system (sic) management solution https://access.redhat.com/articles/2280891 and here https://access.redhat.com/documentation/en-us/red_hat_satellite/6.3/html-single/release_notes/. 

It is also possible that in the future qpc could incorporate other systems management solutions, and it gets awkward to make individual references constantly when you get beyond two things that can be grouped. In short, I'm pretty firm that Satellite + vCenter Server = systems management solution as the generic noun or the generic adjective (*systems management solution* information, when contrasted with *network* information).

What I can do: Those definition paragraphs for the major objects in the man pages are a little creaky by now due to add-ons. I can look at those instances and *maybe* change a few to un-parallel them. So instead of "network information and systems management solution information..." it is possible that a few could be changed to "...network information and information from systems management solutions..." However, I'll just have to look at the unparallel statements and see if they bother me (see my comment about predictable word patterns in my response to a different comment).

Reply:
I went with the "network information, ..., or information about a systems management solution such as ..." for this one.

Reply:
cool